linen_examples/imagenet/README.md

```diff
@@ -10,10 +10,10 @@ This example uses linear learning rate warmup and cosine learning rate schedule.
 ### Supported setups
 The model should run with other configurations and hardware, but explicitely tested on the following.
 
-| Hardware | Batch size | Training time | Top-1 accuracy  | TensorBoard.dev |
-| --- | --- | --- | --- | --- |
-| 8 x Nvidia V100 (16GB)  | 512  |  13h 25m  | 76.63% | [2020-03-12](https://tensorboard.dev/experiment/jrvtbnlETgai0joLBXhASw/) |
-| 8 x Nvidia V100 (16GB), mixed precision  | 2048  | 5h 51m | 76.39% | [2020-03-11](https://tensorboard.dev/experiment/F5rM1GGQRpKNX207i30qGQ/) |
+| Name                    |   Steps | Walltime   | Top-1 accuracy   | Metrics                                                                                                                               | Workdir                                                                                                                                                              |
+|:------------------------|--------:|:-----------|:-----------------|:--------------------------------------------------------------------------------------------------------------------------------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| v100_x8                 |  250199 | 18.0h      | 76.62%           | [tfhub.dev](https://tensorboard.dev/experiment/6yEYynrdRS2HZqHAD2ejPQ/#scalars&_smoothingWeight=0&regexInput=v100_x8)                 | [gs://flax_public/examples/imagenet/v100_x8](https://console.cloud.google.com/storage/browser/flax_public/examples/imagenet/v100_x8)                                 |
```

Comment:
It really is concerning that this got so much slower. Do we have any lead on why?

Reply:
Rerunning it with #717 as suggested by @myagues above... Will update here when the results are ready.

Reply:
A this explains why I could not find anything during profiling. The steps are fast there’s just too many

Reply:
Ok that fixed it :) We're now even faster & more accurate (at least in this run ;-)